app/helpers/publish/value_helper.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+module Publish
+  module ValueHelper
+    def value_provided?(value)
+      value.presence || tag.span(t('value_not_entered'), class: 'govuk-hint').html_safe
```

Comment:
Change `value_not_entered`  to `value_not_entered_html`

So it becomes
```suggestion
      value.presence || t('value_not_entered_html')
```